source/features/clean-repo-sidebar.tsx

```diff
@@ -8,10 +8,43 @@ import features from '.';
 async function init(): Promise<void> {
 	document.body.classList.add('rgh-clean-repo-sidebar');
 
-	// Hide redundant "Releases" section
+	// Hide "Readme" link made unnecessary by toggle-files-button #3580
+	(await elementReady('.repository-content .BorderGrid-row:first-child .muted-link[href="#readme"]', {waitForChildren: false}))?.parentElement!.remove();
+
+	// Remove whitespace in license link to fix alignment of icons https://github.com/sindresorhus/refined-github/pull/3974#issuecomment-780213892
+	const licenseLinkText = (await elementReady('.repository-content .BorderGrid-row:first-child [href*="/license" i]', {waitForChildren: false}))?.childNodes[2];
+	if (licenseLinkText) {
+		licenseLinkText.textContent = licenseLinkText.textContent!.trim();
+	}
```

Comment:
Some repos don't have a license file, so this would wait until dom-ready, which would slow down every other part of the feature. Actually this applies to almost every section, so maybe this init should be split into:

```ts
async function init(): Promise<void> {
	void removeReadmeLink();
	void cleanLicenseText();
	void cleanReleases();
	void removePackages();
	
	await domLoaded;
	// etc, inline here other parts that don't have `await`. `domLoaded` might just be enough since it's near the end of the page anyway
}
```

Reply:
It's definitely much snappier now! :tada: